### ISSUE TITLE
New created playlist are public by default

### ISSUE THREAD

**Issue by MaverickPOL** [jellyfin/jellyfin#12835]
### This issue respects the following points:

- [X] This is a **bug**, not a question or a configuration issue; Please visit our forum or chat rooms first to troubleshoot with volunteers, before creating a report. The links can be found [here](https://jellyfin.org/contact/).
- [X] This issue is **not** already reported on [GitHub](https://github.com/jellyfin/jellyfin/issues?q=is%3Aopen+is%3Aissue) _(I've searched it)_.
- [X] I'm using an up to date version of Jellyfin Server stable, unstable or master; We generally do not support previous older versions. If possible, please update to the latest version before opening an issue.
- [X] I agree to follow Jellyfin's [Code of Conduct](https://jellyfin.org/docs/general/community-standards.html#code-of-conduct).
- [X] This report addresses only a single issue; If you encounter multiple issues, kindly create separate reports for each one.

### Description of the bug

In new version of the Jellyfin i saw that new created playlists are public by default. User must choose to make playlist private.  And in description of 10.9.x version there was information that those playlist should be private by default (https://jellyfin.org/posts/jellyfin-release-10.9.0/)

This is not that issue when i use WEB GUI or Jellyfin app, but when user use for example Finamp, he dont have option to choose that playlist should be private or not, and if playlist by default are public, all created playlist become public.

Could we have option to choose how playlist should be create by default (without user interaction), or just have private by default?
So right now i believe this is bug, becouse in patchnotes was information that playlist should be private by default.

It will be very nice also if there will be option to change public/private for playlist. Maby even from admin page for all users?

### Reproduction steps

Click Create New Playlist for audio file

### What is the current _bug_ behavior?

New created playlist are by default public.

### What is the expected _correct_ behavior?

Correct behaviour should be "private playlist by default without user interaction"

### Jellyfin Server version

10.9.11+

### Specify the build version

10.9.11

### Environment

```markdown
- OS: Debian
- Linux Kernel:  n/a
- Virtualization: LXC
- Clients: jellyfin, finamp
- Browser: chrome
- FFmpeg Version: n/a
- Playback Method: n/a
- Hardware Acceleration: n/a
- GPU Model: n/a
- Plugins: none
- Reverse Proxy: nginx
```

### Jellyfin logs

```shell
[2024-10-17 00:17:20.445 +02:00] [INF] Creating image collage and saving to "/var/cache/jellyfin/temp/3537978bedb841f8a521bf83584d14df.png"
[2024-10-17 00:17:20.641 +02:00] [INF] Completed creation of image collage and saved to "/var/cache/jellyfin/temp/3537978bedb841f8a521bf83584d14df.png"
[2024-10-17 00:18:24.674 +02:00] [INF] Removing item, Type: "Playlist", Name: "newtest", Path: "/var/lib/jellyfin/data/playlists/newtest", Id: f3877a57-9fa7-934e-563a-d8c02b1d3062
[2024-10-17 00:18:24.675 +02:00] [INF] Deleting item path, Type: "Playlist", Name: "newtest", Path: "/var/lib/jellyfin/data/playlists/newtest", Id: f3877a57-9fa7-934e-563a-d8c02b1d3062
```

**Comment by gnattu** [comment]
I think this is a pure client specific design and not server default. If the client does not supply whether the playlist is public the server will create as private. That is the web which implemented this selection to allow public access to match old behavior.

**Comment by MaverickPOL** [comment]
No. I as i mention before if i use Finamp for creating playlist they also are created as public, and there is no option in Finamp to chose.

Jellyfin PC app, WEB GUI or Android App also have public "checkmark" check as default.

**Comment by felix920506** [comment]
Closing as working as intended.

**Comment by felix920506** [comment]
Closing as working as intended.

**Comment by MaverickPOL** [comment]
Intendes as patchnotes say was "default playlist are private" and there are not, default are public. But if change 0 for 1 is so difficult, sure close it.

**Comment by gnattu** [comment]
Well this issue breaks into two parts:

- The Web checking that tick by default, this is pure client side implementation and has nothing to do with the server handling.
- What you mentioned, clients not migrated to 10.9 flavor of API requests which contains ACL info defaulted to create the playlists as public, this is handed by the API controller to make the behavior similar to 10.8 server when the client is using 10.8-like requests. This is also used to workaround the issue on the first few 10.9 releases where the visibility of the playlist cannot be changed after creation. To reverse this is already too late for 10.9. Let's see if we can get 10.10 to remove this workaround so that new playlists created with no ACL info will create private playlists instead.

**Comment by gnattu** [comment]
Alright, we now have a conclusion on this: we decided to remove the pre-10.9 api behavior in 10.11, which is the next next release to give clients 2 releases/1 year to migrate.
